locationdelegation/src/main/java/com/google/androidbrowserhelper/locationdelegation/PermissionRequestActivity.java

```diff
@@ -0,0 +1,112 @@
+// Copyright 2020 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.androidbrowserhelper.locationdelegation;
+
+import android.Manifest;
+import android.content.Context;
+import android.content.Intent;
+import android.content.pm.PackageManager;
+import android.os.Bundle;
+import android.os.Handler;
+import android.os.Looper;
+import android.os.Message;
+import android.os.Messenger;
+import android.os.RemoteException;
+import android.text.TextUtils;
+
+import androidx.appcompat.app.AppCompatActivity;
+import androidx.browser.trusted.TrustedWebActivityCallbackRemote;
+import androidx.core.app.ActivityCompat;
+
+/**
+ * This is a simple transparent activity that will bring up the permission prompt. On either approve
+ * or disapprove, this will send the result via the {@link Messenger} provided with the intent, and
+ * then finish.
+ */
+public class PermissionRequestActivity extends AppCompatActivity {
```

Comment:
Do we not need to include this Activity in a manifest somewhere?

Reply:
So I think we can add it to a manifest for the module, then if this module is included in some app, the manifest is merged together.

Reply:
wow, I didn't realize I can add an empty <application> tag in the manifest. Added now